4_arm.ipynb

```diff
@@ -497,11 +919,22 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "今回は、レゴブロック「e_lego_duplo」をHSRの前方0.4mの位置に出現させます。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "This time, we will let the Lego block \"e_lego_duplo\" appear 0.4m in front of the HSR:"
```

Comment:
"This time, we will make the Lego block \"e_lego_duplo\" appear 0.4m in front of the HSR:"